src/utils/fetchUtils.ts

```diff
@@ -224,3 +231,62 @@ export async function fetchLanguages(): Promise<ILanguage[]> {
   });
   return languages;
 }
+
+function sortByHourAndMin(a: IEvent, b: IEvent) {
+  const aStart = a.dateStart;
+  const bStart = b.dateStart;
```

Comment:
Maybe deconstruct `dateStart` already in the function declaration?

Reply:
👍